guide/foundations/backups.md

```diff
@@ -0,0 +1,265 @@
+---
+layout: guide
+title: Bitcoin backups
+description: A beginner's guide to Bitcoin backups.
+nav_order: 11
+parent: Foundations
+permalink: /guide/foundations/backups/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/foundations/wallet-interoperability/wallet-interoperability-preview.jpg
+---
+
+<!--
+
+Editor's notes
+
+A guide meant to lower the barrier for first time users of self-custody wallets.
+
+Illustration sources
+
+// 
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/foundations/backups/backups.jpg"
+   retina = "/assets/images/guide/foundations/backups/backups@2x.jpg"
+   mobile = "/assets/images/guide/foundations/backups/backups-mobile.jpg"
+   mobileRetina = "/assets/images/guide/foundations/backups/backups-mobile@2x.jpg"
+   alt-text = "Several devices exchanging data"
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+# Bitcoin backups
+
+The self-custody of bitcoin is an important right, but also a financial responsibility that can take practice getting used to, and scale from pocket money to life savings.
+
+Making a secure backup of your wallet should be straightforward. This guide is meant to provide a starting point for simple bitcoin backups. From a basic setup, you can increase the security level as your funds grow. 
+
+
+## What are backups?
+
+When we talk about backups, we mean saving the information needed to restore access to a wallet outside of the application that created the private keys. Most often, this will be the *recovery-phrase* of of 12 or 24 words, but can also include other information like a *pass-phrase*.
```

Comment:
*recovery-phrase* would be good to link to glossary `guide/glossary/#recovery-phrase`

Reply:
Not yet mentioning derivation path, but will look into it for the template and consider how/if to talk about it on this page.

Reply:
Now mention briefly, can expand in the future.